client/src/client/client.rs

```diff
@@ -45,10 +45,10 @@ pub trait Client<C: ClientHandle> {
     /// get a mutable reference to the tokio Core associated to the Client
     fn get_io_loop(&self) -> RefMut<Core>;
 
-    /// Get a mutable handle reference tot the Core assiated to the Client
+    /// Get a mutable handle reference tot the Core associated to the Client
     fn get_client_handle(&self) -> RefMut<C>;
 
-    /// A *classic* DNS query, i.e. does not perform and DNSSec operations
+    /// A *classic* DNS query, i.e. does not perform any DNSSec operations
     ///
     /// *Note* As of now, this will not recurse on PTR or CNAME record responses, that is up to
```

Comment:
since you're making this change, would you mind removing the CNAME info here? It's still accurate about PTR, but CNAME recursion is now supported.